BLOsc error build and install on Linux

Hi,
I'm unable to install the plugins because CMake tells me that BLOsc was not declared during the building.
This is the text of the error:

`In file included from /home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp:4:
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.hpp:26:18: error: ‘BlOsc’ in namespace ‘daisysp’ does not name a type
   26 |         daisysp::BlOsc blosc;
      |                  ^~~~~
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp: In constructor ‘BLOsc::BLOsc::BLOsc()’:
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp:14:3: error: ‘blosc’ was not declared in this scope
   14 |   blosc.Init(samplerate);
      |   ^~~~~
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp: In member function ‘void BLOsc::BLOsc::next(int)’:
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp:32:5: error: ‘blosc’ was not declared in this scope
   32 |     blosc.SetFreq(slopedFreq.consume());
      |     ^~~~~
gmake[2]: *** [CMakeFiles/BLOsc_scsynth.dir/build.make:76: CMakeFiles/BLOsc_scsynth.dir/plugins/BLOsc/BLOsc.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1898: CMakeFiles/BLOsc_scsynth.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2`

Any solutions?
Thank you, Enrico :)

**Desktop:**
 - OS: Fedora Linux 39
 - Architecture: x86
 - SC Version: 3.14.0-dev
 - PortedPlugins Version: v0.4.1

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -676,7 +676,7 @@ if(USE_DAISYSP_LIB)
 	CPMAddPackage(
 		NAME daisysp
 		GITHUB_REPOSITORY "electro-smith/DaisySP"
-		GIT_TAG master
+		GIT_TAG v0.0.2
 		)
 
 	include_directories("include/daisysp")

